_source/logzio_collections/_security-sources/crowdstrike.md

```diff
@@ -139,19 +141,14 @@ Upon deployment, the Crowdstrike connector connects to your Crowdstrike account
    </label>
    ```
 
-   <!-- info-box-start:info -->
-   The default configuration for the log level is set to **info**. The **log_level** setting defines which events are recorded in the log. In order of verbosity, the log level can be defined as **fatal**, **error**, **warn**, **info**, **debug** or **trace**. The **info** level records all events categorized as **info** and higher in verbosity. To learn more about the settings used in the configuration file, see [Logz.io plugin for Fluentd](https://github.com/logzio/fluent-plugin-logzio).
-   {:.info-box.note}
-   <!-- info-box-end -->
-
-   <!-- info-box-start:info -->
+   The default configuration for the log level is set to **info**. The **log_level** setting defines which events are recorded in the log. In order    of verbosity, the log level can be defined as **fatal**, **error**, **warn**, **info**, **debug** or **trace**. The **info** level records all    events categorized as **info** and higher in verbosity. To learn more about the settings used in the configuration file, see [Logz.io plugin for    Fluentd](https://github.com/logzio/fluent-plugin-logzio).
```

Comment:
This concept explanation breaks the flow of a procedure.  
Can it be moved into the overview?

Reply:
Thanks for the feedback Yael, I've moved it to the beginning of the step.